app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Users</h1>
+<ul class='users-list'>
+<% @users.each do |user| %>
+  <li>
+<%= render partial: "username", locals: { user: user } %>
+</li>
+<% end %>
+</ul>
```

Comment:
- [ ] Please make sure clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)